### ISSUE TITLE
Component not updating

### ISSUE THREAD

**Issue by wenliangz** [emeeks/semiotic#352]
Thank you for the wonderful package.I am facilitated by the plots and here below is my first test try.  If I feed the component with data directly within `render()`, it renders without any problems. However, if I feed to XYframe component with State data like below, it does not work. I don't see any plot coming out. I tried many different ways. Now I am wondering if it is because that after the component first rendered with empty data, it never updates when there is a new data from the state?  Please help. Thanks! 

```
import React, { Component } from 'react';

class ScatterPlot2 extends Component {

    constructor(props){
        super(props);
        this.state = {
            data: [],
        };

    }

  componentDidMount(){
      this.setState({
          data: [{x:150,y:110},{x:120,y:120}],
      });
  }

  render() {
      const {data} = this.state;
      // const data = [{x:100,y:130},{x:120,y:140}]
      console.log(data)
      const frameProps ={
            title: "Scatter Plot",
            size: [1000,500],
            points: data,
            xAccessor: "x",
            yAccessor: "y",
            pointStyle: { fill: "red", stroke: "blue", strokeWidth: 5 },
            margin: {top: 50, left: 50, bottom: 50, right: 50},
            axes: [{ orient: "left"}, {orient: "bottom"}]
          };
          return (
                <XYFrame {...frameProps} />
              )
     }

}

export default ScatterPlot2;
```

**Comment by emeeks** [comment]
Thanks for finding this! It will be fixed in the next version.

**Comment by wenliangz** [comment]
Thank you so much! I will be looking forward to the update!

**Comment by emeeks** [comment]
Fixed in `1.15.9`
